pkg/apis/servicecatalog/checksum/checksum_test.go

```diff
@@ -64,3 +64,19 @@ func TestBindingSpecChecksum(t *testing.T) {
 		t.Fatalf("versioned and unversioned checksums should match; expected %v, got %v", e, a)
 	}
 }
+
+func TestBrokerSpecChecksum(t *testing.T) {
+	spec := servicecatalog.BrokerSpec{
+		URL:      "https://kubernetes.default.svc:443/brokers/template.openshift.io",
```

Comment:
I'll get rid of it. I just happened to have the template broker deployed to look at the exact syntax for the URL.